src/Spice86.Core/Emulator/CPU/CPU.cs

```diff
@@ -1019,7 +1019,7 @@ private static bool IsStringOpUpdatingFlags(int stringOpCode)
                 break;
             case 0xF4:
                 // HLT
-                _loggerService.Information("HLT instruction encountered, halting!");
+                _loggerService.Verbose("HLT instruction encountered, halting!");
```

Comment:
I'd think this needs to be at least Information, maybe even Warning

Reply:
Implemented.